e2e/test/prerequisites/E2ETestsSetup/e2eTestsSetup.ps1

```diff
@@ -620,25 +598,7 @@ if ($groupEnrollmentExists)
     az iot dps enrollment-group delete -g $ResourceGroup --dps-name $dpsName --enrollment-id $groupEnrollmentId
 }
 Write-Host "`nAdding group enrollment $groupEnrollmentId."
-az iot dps enrollment-group create -g $ResourceGroup --dps-name $dpsName --enrollment-id $groupEnrollmentId --ca-name $uploadCertificateName --output none
-
-$individualEnrollmentId = "iothubx509device1"
-$individualDeviceId = "provisionedx509device1"
-$individualEnrollmentExists = az iot dps enrollment list -g $ResourceGroup  --dps-name $dpsName --query "[?deviceId=='$individualDeviceId'].deviceId" --output tsv
-if ($individualEnrollmentExists)
-{
-    Write-Host "`nDeleting existing individual enrollment $individualEnrollmentId for device $individualDeviceId."
-    az iot dps enrollment delete -g $ResourceGroup --dps-name $dpsName --enrollment-id $individualEnrollmentId
-}
-Write-Host "`nAdding individual enrollment $individualEnrollmentId for device $individualDeviceId."
-az iot dps enrollment create `
-    -g $ResourceGroup `
-    --dps-name $dpsName `
-    --enrollment-id $individualEnrollmentId `
-    --device-id $individualDeviceId `
-    --attestation-type x509 `
-    --certificate-path $individualDeviceCertPath `
-    --output none
```

Comment:
DPS X509 self-signed certificates are being generated on-demand, so this can be removed